animeal/src/Modules/Login/View/Utils/LoginViewActionMapper.swift

```diff
@@ -7,13 +7,14 @@
 
 import Foundation
 
+// sourcery: AutoMockable
 protocol LoginViewActionMappable {
-    func map(_ input: LoginModelAction) -> LoginViewAction
-    func map(_ input: [LoginModelAction]) -> [LoginViewAction]
+    func mapAction(_ input: LoginModelAction) -> LoginViewAction
+    func mapActions(_ input: [LoginModelAction]) -> [LoginViewAction]
```

Comment:
Yes, in case of method overloading Sourcery generated mocks with equal property names, and it resulted in  compilation error, so we need to avoid using such names in future.

Reply:
Because of problems with missing SPM I recreated this PR, please check https://github.com/AnimealProject/animeal_iOS/pull/16